api/wasm-interpreter/src/lib.rs

```diff
@@ -37,13 +37,23 @@ impl CompilationResult {
 const IMPORT_CALLBACK_FUNCTION_SECTION: &'static str = r#"
 type ImportCallbackFunction = (url: string) => Promise<string>;
 "#;
+#[wasm_bindgen(typescript_custom_section)]
+const CURRENT_ITEM_INFORMATION_CALLBACK_FUNCTION_SECTION: &'static str = r#"
+type CurrentItemInformationCallbackFunction = (url: string, start_line: number, start_column: number, end_line: number, end_column: number) => void;
```

Comment:
can't the two string be merged in one?
Otherwise we can keep the string next to the struct.